vault/core_test.go

```diff
@@ -148,6 +149,103 @@ func TestCore_Unseal_MultiShare(t *testing.T) {
 	}
 }
 
+// TestCore_UseSSCTokenToggleOn will check that the root SSC
+// token can be used even when disableSSCTokens is toggled on
+func TestCore_UseSSCTokenToggleOn(t *testing.T) {
+	c, _, root := TestCoreUnsealed(t)
+	c.disableSSCTokens = true
+	req := &logical.Request{
+		Operation: logical.UpdateOperation,
+		Path:      "secret/test",
+		Data: map[string]interface{}{
+			"foo":   "bar",
+			"lease": "1h",
+		},
+		ClientToken: root,
+	}
+	ctx := namespace.RootContext(nil)
+	resp, err := c.HandleRequest(ctx, req)
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+	if resp != nil {
+		t.Fatalf("bad: %#v", resp)
+	}
+
+	// Read the key
+	req.Operation = logical.ReadOperation
+	req.Data = nil
+	req.SetTokenEntry(&logical.TokenEntry{ID: root, NamespaceID: "root", Policies: []string{"root"}})
```

Comment:
I'm not really sure, but it seems to be the pattern for the other tests in this file before calling `c.HandleRequest`. I believe that since `c.HandleRequest` is a vault internal function, there's some initial parsing of the request that is skipped.

Reply:
Ah, indeed, it's handleCancelableRequest that calls PopulateTokenEntry.  Maybe you could do the same instead of hardcoding this?